repose-aggregator/components/filters/client-auth/src/main/java/org/openrepose/filters/clientauth/ClientAuthenticationHandlerFactory.java

```diff
@@ -77,7 +78,12 @@ public void configurationUpdated(ClientAuthConfig modifiedConfig) {
                     accountRegexExtractor.addPattern(clientMapping.getIdRegex());
                 }
                 if (modifiedConfig.getAtomFeeds() != null) {
-                    activateOpenstackAtomFeedListener(modifiedConfig);
+                    try {
+                        activateOpenstackAtomFeedListener(modifiedConfig);
+                    } catch (TimeoutException e) {
```

Comment:
Upon further review of this, the NPE was being caught at `PowerApiConfigurationManager.subscribeTo()` (ln:123) and `PowerApiUpdateManagerEventListener.onEvent()` (ln:49) depending on exactly when in the Repose life-cycle it was thrown. In both cases the catching of the NPE is what triggers marking the filter as Failed To Load before eventually logging the NPE.

To consume it here runs the risk of the filter not being marked as Failed To Load. If we don't throw an un-checked exception, then we have to burden all of the existing implementations with guarding against the possibility of a new checked exception being thrown.

I can add a new `UncheckedAkkaServiceClientException` that extends `RuntimeException` and have it wrap the `AkkaServiceClientException` or it's `Cause` in order to let it bubble to the top?

I heard you like wrappers for your wrappers, because wrapping is good!?!?